### DISCUSSION TITLE
Team member can checkin tickets without permission. Bug?

### DISCUSSION THREAD

**Discussion by nilskamm** [pretix/pretix#2475] [Q&A] [answered]
Hey pretix heroes!
I´m not sure if i found a bug and want to ask you!
We are using the actual stable docker version of pretix with MySQL 8.0.

I created a new team which shall help during the ticket selling.
They only have the following permissons:
- Can view orders
- Can change orders
- Can view vouchers
- Can change vouchers

Then i logged in with a test account from this team (cache cleaned and using private window in Firefox) and made a checkin which worked, even though the permission "Can perform check-ins" is not activated.
Does it come automatically from one of the other permissions and it is not written in the documentary, is this a bug and can anyone see the same effect?

Thanks a lot!

**Comment by raphaelm** [comment, marked as answer]
This is because of "can change orders" and without checking I'm 90% sure it's documented in the description of the permissions in the backend.

**Comment by nilskamm** [comment]
Yes! You are totally right! I put the user guide on my second screen and didn´t read the description in the backend itself. "facepalm"
